caches/schema-29.0.0.json

```diff
@@ -733,7 +733,7 @@
               "properties": {
                 "action": { "type": "string" },
                 "data": {
-                  "type": "object",
+                  "type": ["array", "object"],
                   "properties": {
```

Comment:
Does `properties` work with an attribute that can either be an object or an array? I wasn't able to find any other similar scenario.